server/bin/pbench-base.sh

```diff
@@ -104,11 +104,8 @@ function log_finish {
 
```

Comment:
I am not entirely sure this works.  Can you prove that this is correct for me so that I don't have to worry about it?

The current code properly closes the two files, but 200 FD is just copy of the original 2 FD, so keeping it open does not hurt anything.  And typically we are going to exit soon after `log_finish` so I am not sure it is worth making this subtle and elegant change that still makes me nervous.  =)

Reply:
> Can you prove that this is correct for me so that I don't have to worry about it?

I'm not sure that that is possible.  (And, what I think you're really asking is that I prove that it doesn't not work...and, as we know, "proving a negative" is notoriously difficult.)

If you don't want to try the ["Moving File Descriptors" syntax](https://linux.die.net/man/1/bash#:~:text=Moving%20File%20Descriptors), which is quite understandable in the context of `b0.69`, there's a middle ground, which is to add this line
```
    exec 200>&-  # Close stderr copy
```
between these:
```
    exec 100>&-  # Close log file
    exec 4>&-    # Close error file
```
to address the omission.

...or, as you say, we could just ignore it.

Reply:
Pulled these changes into a separate PR, #3304.